mdakit_membcurv/tests/datafiles.py

```diff
@@ -0,0 +1,23 @@
+
+
+"""
+Location of datafiles for Membrane Curvature unit tests
+=======================================================
+
+MD simulations files stored in ``\\data`` sub-directory.
+
+    from mdakit_membcurv.datafiles import *
+"""
+
+__all__ = [
+    "GRO_",  # Gromacs file of POPC POPE CHOL membrane
+    "XTC_"  # Gromacs trajectory of 10 frames.
```

Comment:
Maybe be more explicit and name them `GRO_MIXED_MEMBRANE` and `XTC_MIXED_MEMBRANE` instead of a trailing underscore?

Reply:
Thanks, changed to a more explicit name.